r2i/onnxrt/engine.h

```diff
@@ -50,19 +64,27 @@ class Engine : public IEngine {
   size_t output_size;
   size_t num_input_nodes;
   size_t num_output_nodes;
+  Ort::Env env {nullptr};
```

Comment:
These are not pointers. Does this still reflect the initial state you want for these members?